tasgrid/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+########################################################################
+# tasgrid command line tool
+########################################################################
+
+########################################################################
+# add the tasgrid executable
+########################################################################
+add_executable(Tasmanian_tasgrid tasgrid_main.cpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/TasmanianSparseGrid.hpp
```

Comment:
The TasmanianSparseGrid file is not associated with this target. The file is used as an external dependence and doesn't need to be added to the sources.

This is in contrast with CLICommon which is a file used by all CLI tools.

Reply:
Fixed.